### ISSUE TITLE
Image filtering on OpenImage

### ISSUE THREAD

**Issue by stalkerrush** [JudyYe/shelf-sup-mesh#1]
Hi @JudyYe , thanks for releasing your great work! 

Could you please provide more details on how you created the split for OpenImage? You mentioned a fine-tuned classifier is used to remove occluded or truncated images, but what method are you using here and how did you fine-tune it? (I notice there are occlusion and truncation annotation for boxes, did you train on these and inference on the cropped images? If so, why not directly use the annotation...) Also, the link to the file including your filtered split seems not working.

Thank you.

**Comment by JudyYe** [comment]
Hi @stalkerrush,

Apologies on the late response. 

1. OpenImages split: we use their official split. 
2. Why not directly use annotation?  and how to finetune? Although OpenImages provides a good annotation of masks, it does not specify whether these annotated masks are occluded or truncated. Therefore we manually label 50 images for 50 categories and finetune a ResNet to classify occlusion/truncation. 
3. link: sorry about that and thank you for pointing it out. The link to Openimages [models](https://drive.google.com/file/d/1BOIcPRXdpKPNi6G3pjDSkyaKGCf5kMDm/view?usp=sharing) and [split with truncation score](https://drive.google.com/file/d/1QIgr3_OYQKC1xtMsHTm-XdYLuMCxlmdA/view?usp=sharing) are updated. The score is in the last column whose -1 indicates non inferred and 0-1 indicates predicted score.
